[2024-09-16T15:37:35.118+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-16T15:37:35.159+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_alchemy.say_welcome_alchemy manual__2024-09-16T15:37:33.849051+00:00 [queued]>
[2024-09-16T15:37:35.181+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_alchemy.say_welcome_alchemy manual__2024-09-16T15:37:33.849051+00:00 [queued]>
[2024-09-16T15:37:35.183+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-16T15:37:35.210+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): say_welcome_alchemy> on 2024-09-16 15:37:33.849051+00:00
[2024-09-16T15:37:35.222+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1146) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-16T15:37:35.225+0000] {standard_task_runner.py:72} INFO - Started process 1147 to run task
[2024-09-16T15:37:35.224+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_alchemy', 'say_welcome_alchemy', 'manual__2024-09-16T15:37:33.849051+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/test_alchemy_dags.py', '--cfg-path', '/tmp/tmppqohjz55']
[2024-09-16T15:37:35.227+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask say_welcome_alchemy
[2024-09-16T15:37:35.318+0000] {task_command.py:467} INFO - Running <TaskInstance: test_alchemy.say_welcome_alchemy manual__2024-09-16T15:37:33.849051+00:00 [running]> on host e54c4d73c780
[2024-09-16T15:37:35.507+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='muhii.smta@gmail.com' AIRFLOW_CTX_DAG_OWNER='Muhii' AIRFLOW_CTX_DAG_ID='test_alchemy' AIRFLOW_CTX_TASK_ID='say_welcome_alchemy' AIRFLOW_CTX_EXECUTION_DATE='2024-09-16T15:37:33.849051+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-16T15:37:33.849051+00:00'
[2024-09-16T15:37:35.509+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-16T15:37:35.542+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-16T15:37:35.544+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hallo !"']
[2024-09-16T15:37:35.565+0000] {subprocess.py:86} INFO - Output:
[2024-09-16T15:37:35.568+0000] {subprocess.py:93} INFO - Hallo !
[2024-09-16T15:37:35.570+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-16T15:37:35.625+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-16T15:37:35.626+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=test_alchemy, task_id=say_welcome_alchemy, run_id=manual__2024-09-16T15:37:33.849051+00:00, execution_date=20240916T153733, start_date=20240916T153735, end_date=20240916T153735
[2024-09-16T15:37:35.686+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-16T15:37:35.746+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-16T15:37:35.751+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
